doc/InfiniTimeVision.md

```diff
@@ -0,0 +1,32 @@
+# InfiniTime Vision
+
+The purpose of this document is to steer efforts towards a common goal, and as such should be taken into consideration with all developments.
+
+## InfiniTime
+
+InfiniTime is a community-built smartwatch firmware, providing freedom and privacy that's usually insufficient in wearable technology.
+
+InfiniTime is not to be used for medical or other health tracking purposes.
+
+## Core Principles
+
+- KISS (Keep It Short and Simple)
+- Reliability
+- Battery efficiency
+- Easy and simple navigation
+- Behaviour should be predictable and easy to understand
+- Prefer solid default experience over customization
+- Personalization is achieved through custom watchfaces.
```

Comment:
Currently watchfaces are really hard to implement. Designers can't implement them because most of them can't program, and developers are predominantly terrible at UI/UX.

Once InfiniTime has external storage and this kind of stuff I'll probs work on a specification to make the creation of watchfaces easy & flexible. I'll harness my experience as a product engineer, and my close collaboration with designers in my previous work.

Reply:
> Personalization is achieved through custom watchfaces.

Also, I believe that it contradicts the statement `capability to upload apps`.

Reply:
> Currently watchfaces are really hard to implement. Designers can't implement them because most of them can't program, and developers are predominantly terrible at UI/UX.

Makes sense. It would be great to get some collaboration happening between designers and developers.

> Also, I believe that it contradicts the statement capability to upload apps.

Hmm, how so?

Reply:
> Makes sense. It would be great to get some collaboration happening between designers and developers.

Great! I'll work on mockups before implementing it so as to make sure we are on the same page. :)

> Hmm, how so?

I interpret this statement as exhaustive about how we can customize the OS. What do you think? It does not explicitly state “only” but many users and contributors won't pick the subtlety of the wording.

Do we plan to add other customizations than watchfaces and apps?

If there are not too many, I'd suggest making this list exhaustive.

Reply:
Ah I see what you mean. Here personalization means aesthetic, non-functional options, like changing fonts, certain colors etc. We've decided to limit aesthetic customization mainly to the watchface to keep things simple and try to provide a great user experience by default.

Many projects end up making a lot of things configurable, which quickly becomes overwhelming for users. We try to avoid adding too many options if possible, but not to the point of detriment.